app/models/annotation/nml/NmlParser.scala

```diff
@@ -156,11 +156,11 @@ class NmlParser @Inject()(datasetDAO: DatasetDAO) extends LazyLogging with Proto
         taskBoundingBox: Option[BoundingBox] = if (sharedParsingParameters.isTaskUpload)
           parseTaskBoundingBox(parameters \ "taskBoundingBox")
         else None
-        userBoundingBoxes = parseBoundingBoxes(parameters \ "userBoundingBox")
       } yield {
+        var userBoundingBoxes = parseBoundingBoxes(parameters \ "userBoundingBox")
         if (!sharedParsingParameters.isTaskUpload) {
           parseTaskBoundingBoxAsUserBoundingBox(parameters \ "taskBoundingBox", userBoundingBoxes)
-            .map(asUserBoundingBox => userBoundingBoxes :+ asUserBoundingBox)
+            .foreach(asUserBoundingBox => userBoundingBoxes = userBoundingBoxes :+ asUserBoundingBox)
```

Comment:
The changes to `userBoundingBoxes` weren't saved. These changes now work close to what the logic was before the #8075 pr. cf: https://github.com/scalableminds/webknossos/blob/5d3d66d2ae426737261dc1714792730969ec8d72/app/models/annotation/nml/NmlParser.scala#L79-L83